### DISCUSSION TITLE
Postman VSCode Extension

### DISCUSSION THREAD

**Discussion by marcozanotti** [posit-dev/positron#5099] [Q&A] [answered]
I tried to install the [Postaman VSCode Extension](https://github.com/postmanlabs/postman-app-support) but it is not working. 
Is there any chance that this extension can work with Positron?
Thanks

**Comment by sharon-wang** [comment, marked as answer]
Hey @marcozanotti! I'm also seeing an error after installing the Postman extension in Positron.

<img width="461" alt="image" src="https://github.com/user-attachments/assets/ea408459-661d-4720-97fa-3d8d73935cfc">

The error message is coming from the Postman Extension, so I suspect they have a list of supported IDEs, likely the ones mentioned in the extension README:

<img width="400" alt="image" src="https://github.com/user-attachments/assets/01c619fd-68a7-4a93-908c-4b0285c13338">

I see you already opened [an issue](https://github.com/postmanlabs/postman-app-support/issues/13218) on the Postman side to allow support in Positron -- that would be my suggestion, so you're already a step ahead :)

**Reply by marcozanotti** [reply]
Thank you for your feedback. So wait for them!
